content/swift/concepts/arrays/terms/isEmpty/isEmpty.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.isEmpty'
+Description: 'Returns true if an array contains no elements, returns false if the array contains one or more elements.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.isEmpty`** method returns true if an array contains no elements, returns false if the array contains one or more elements.
+
+## Syntax
+
+```pseudo
+arrayName.isEmpty
+```
+
+## Example
+
+```swift
+var favoriteSongs = [String]()
+
+print(favoriteSongs.isEmpty)
+//Output: true
+
+favoriteSongs.append("Stayin' Alive")
+print(favoriteSongs.isEmpty)
+//Output: false
```

Comment:
why don't you create a output section for the outputs. Something like this
<img width="457" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/0ca0732e-3efa-47d8-86f3-04e4b51e9c05">

Reply:
I will make this change. Thank you for the feedback!